### DISCUSSION TITLE
Example project has no output on Mac OS 13

### DISCUSSION THREAD

**Discussion by cheesymoo** [premake/premake-core#2026] [Q&A]
I downloaded the binary of Premake of MacOSX and copied them to my usr/local/bin and usr/local/lib directories.
I can call premake5 from Terminal succesfully.

I recreated this example: https://premake.github.io/docs/your-first-script

When I run:
`premake5 vs2013`

I get:

```
Building configurations...
Running action 'vs2013'...
Done (16ms).
```
But there are no files generated within my directory (or anywhere else I think to look)...

If I replace `vs2013` with `xcode4` I get similar results:

```
premake5 xcode4                                
Building configurations...
Running action 'xcode4'...
Done (18ms).
```

Except there are no project files generated! There are no warnings or errors displayed.

What could be going wrong? Would the files be output somewhere else? I have configured nothing at this point, just doing the Hello World tutorial...

**Comment by nickclark2016** [comment]
If you're on the latest beta, can you revert a version and see if there is still no output?

**Reply by cheesymoo** [reply]
I just tried it with Premake5 alpha 16 with the same results.
Is there anyway to activate verbose logging, or a log elsewhere that would have more output?
I didn't see any system level logs when running this command, apart from the command itself.

**Reply by nickclark2016** [reply]
I'd follow @Jarod42's advice then and check your directory permissions. I don't have an Apple product to test on, so I can't attempt to reproduce locally.

**Comment by Jarod42** [comment]
You should have something like:
```
premake5 xcode4                                
Building configurations...
Running action 'xcode4'...
Generated HelloWorld.xcworkspace/contents.xcworkspacedata...
Generated HelloWorld.xcodeproj/project.pbxproj...
Done (18ms).
```

Note that 'Generated %file' only happens when file is written (and different than the older one).
Generated %file is relative to current directory.
As it is missing for you, no files has been generated :/ .

Do you have write permissions in your directory?

**Reply by cheesymoo** [reply]
Yes, I have r+w perms for the directory.
I even ran the command as sudo and got the same output.

**Reply by Jarod42** [reply]
I test several projects without issues [on mac](https://github.com/Jarod42/premake-sample-projects/actions/workflows/premake5-mac-xcode4.yml).

I don't see typos in example (but not tested it BTW)...

I don't see other reasons for not creating the files :/
